docs/src/userguide/migrating_to_cy30.rst

```diff
@@ -264,6 +264,11 @@ When running into an error it is required to add the corresponding operator::
         Example operator++(int)
         Example operator--(int)
 
+Public Declarations in C++
+==========================
+
+Public declaration in C++ mode are exported as C++ API in Cython 3. This behavior can be changed by
+setting export keyword using ``__PYX_EXTERN_C`` macro: ``-D__PYX_EXTERN_C=extern "C"``.
```

Comment:
1) "to allow Cython module to be implemented in C++ but callable in C."? Just a sentence to explain why you might do it.

2) I wonder if `-D` is the best way to do this. Could it be done with a compiler-independent distutils command

Reply:
Done